src/js/common/browserInfoHelper.js

```diff
@@ -78,20 +90,59 @@ const getFormattedOffset = () => {
     return formattedUTC;
 }
 
+/**
+ * The function returns users timezone offset relative to UTC
+ * @returns {string} UTC concatenated with users broesers offset 
+ */
 export const getTimezone = () => `UTC${getFormattedOffset()}`;
+
+/**
+ * The function returns validated screens width value
+ * @returns {string | null} valid value of screens width 
+ */
 export const getScreenWidth = () =>
   validateAndGetScreenDetail(globalsUtil.getScreen().width);
+
+/**
+ * The function returns validated screens height value
+ * @returns {string | null} valid value of screens height 
+ */
 export const getScreenHeight = () =>
   validateAndGetScreenDetail(globalsUtil.getScreen().height);
+
+/**
+ * The function returns users validated device pixel ratio
```

Comment:
same again (I won't bother commenting on all of them; the comments in this format are the ones I'd like changed)